README.md

```diff
@@ -25,11 +25,11 @@ specified as ``fof(name, axiom, F)`` and ``fof(name, conjecture, F)`` respective
 ### Intuitionistic problems
 The collection of intuitionistic problems is described in [4] and it can be found in the directory  ``ILL``. The problems  were obtained from three main sources:
 
-1. Kleene's problems for intuitionistic logic from "Introduction to Metamathematics"
+1. Kleene's theorems for classic and intuitionistic logic from "Introduction to Metamathematics" (New York: van Nostrand, 1952)
```

Comment:
This is a list of where the intuitionistic problems come from, so shouldn't we only mention Kleene's intuitionistic problems?